l10n/AI_TRANSLATE.sh

```
@@ -0,0 +1,150 @@
+echo "Translation in progress for $LANG_CODE ($LANG_FULLNAME)."
+
+# -------------------- Create directory --------------------
+mkdir -p "$LANG_CODE"
+TRANSLATED_FILE="$LANG_CODE/koreader.po"
+UNTRANSLATED_FILE="$LANG_CODE/untranslated.po"
+UPDATED_TRANSLATED_FILE="$LANG_CODE/updated_translated.po"
+
+INPUTFILE=
+OUTPUTFILE=
+
+if [[ ! -f "$TRANSLATED_FILE" && ! -f "$UNTRANSLATED_FILE" ]] then
```

Shellcheck (0.10.0)

[warning] Use semicolon or linefeed before 'then' (or quote to make it literal).

(SC1010)